.github/mergify.yml

```
@@ -23,6 +23,24 @@ pull_request_rules:
     actions:
       queue:
 
+  - name: Conventional Commit PR title
+    conditions:
+      - base=master
+    actions:
+      post_check:
+        success_conditions:
+          - "title~=^(fix|feat|docs|refactor|chore|deps)(?:\\(.+\\))?:"
+        title: |
```

```suggestion
          - "title~=^(fix|feat|docs|refactor|chore|deps)(?:\\(.+\\))?!?:"
        title: |
```
I think we want to allow `!` to mark breaking changes, right?

---

Ah you are right! Thanks!